server/data/contactsApiClient.ts

```diff
@@ -43,4 +45,25 @@ export default class ContactsApiClient extends RestClient {
       user,
     )
   }
+
+  async searchContact(
+    contactSearchRequest: ContactSearchRequest,
+    user: Express.User,
+    pagination?: PaginationRequest,
+  ): Promise<Contact> {
+    const paginationParameters = pagination ?? { page: 0, size: config.apis.prisonerSearchApi.pageSize || 20 }
```

Comment:
Is it right to use `config.apis.prisonerSearchApi.pageSize` for this? Also, the ticket says 10....

Reply:
PaginationRequest  removed and `import Pageable = contactsApiClientTypes.Pageable` added